docs/getting-started/proofs-and-verification/proofs.md

```diff
@@ -44,4 +47,54 @@ message GetDataProofResponse {
 message MPTrieProofElement {
   repeated bytes hashes = 1;
 }
-```
\ No newline at end of file
+```
+
+### Proofs
+
+#### Ledger connectivity proof
+Proof generation:
+- Zero step – user provided with receipt (block header) from another user who claim that some block is part of ledger
```

Comment:
just use a numbered list, this is awkward

Reply:
Maybe use 'Alice' and 'Bob' to tell the story?

Reply:
user (Alice) is provided with a receipt (block header signed by the server) from another user (Bob) who claim that some block is part of ledger

Reply:
ok